src/components/facets/facet.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FacetsModule } from './facets.module';
+import { Facet } from './models/facet';
+
+@Component({
+    selector: 'app',
+    template: `
+            <ux-facet-container>
+                <ux-facet-check-list header="Authors" [facets]="facets" [id]="chosenID"></ux-facet-check-list>
+            </ux-facet-container>
+    `
+})
+export class FacetCheckListTestComponent {
+
+    facets: Facet[] = [];
+    chosenID: string = 'my-facet';
+
+    constructor() {
+
+        this.facets.push(new Facet('Allen Lucas'));
+        this.facets.push(new Facet('Austin Allison'));
+        this.facets.push(new Facet('Bertie Manning'));
+
+        // sort the users alphabetically
+        this.facets.sort((facetOne, facetTwo) => {
```

Comment:
Remove this, it's irrelevant to the test and does nothing because they are already in alphabetical order.